service/history/queue_factory_base_test.go

```diff
@@ -52,32 +50,17 @@ import (
 	"go.temporal.io/server/service/history/workflow"
 )
 
-// TestQueueModule_ArchivalQueueCreated tests that the archival queue is created if and only if the static config for
-// either history or visibility archival is enabled.
+// TestQueueModule_ArchivalQueue tests that the archival queue is created if and only if the task category exists.
 func TestQueueModule_ArchivalQueue(t *testing.T) {
 	for _, c := range []moduleTestCase{
 		{
-			Name:                "Archival completely disabled",
-			HistoryState:        carchiver.ArchivalDisabled,
-			VisibilityState:     carchiver.ArchivalDisabled,
+			Name:                "Archival enabled",
```

Comment:
nit: this one is `Archival disabled` case?

Reply:
Thanks, fixed